ocr_service/main.py

```
@@ -0,0 +1,55 @@
+logger.info(f"GEMINI_MODEL: {os.environ.get('GEMINI_MODEL', 'Not found')}")
+logger.info(f"DEBUG setting: {os.environ.get('DEBUG', 'Not found')}")
+logger.info("==== End Environment Debug ====")
+
+# Import API endpoints
+from api.endpoints.ocr import router as ocr_router
+from api.endpoints.extraction import router as extraction_router
+from core.config import settings
+
+# Debug: Print settings values
+logger.info("==== DEBUG: Settings Values ====")
+logger.info(f"settings.GEMINI_API_KEY exists: {'Yes' if settings.GEMINI_API_KEY else 'No'}")
```

Ruff (0.8.2)

Logging statement uses f-string

(G004)